packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -68,6 +68,12 @@ export interface PipelineProps {
    */
   readonly artifactBucket?: s3.IBucket;
 
+  /**
+   * The IAM role to be assumed by this Pipeline.
```

Comment:
Can we be a little more precise here? "The IAM role used for executing this Pipeline. If not specified, a new IAM role, trusting the CodePipeline service principal, will be created".